modules/sc-mesh-secure-deployment/configure.sh

```diff
@@ -234,10 +234,8 @@ function server {
   popd
   # Advertise the server using avahi (zeroconf)
   avahi-publish-service mesh_server _http._tcp 5000 &
-  #Install certificate
```

Comment:
we should do chmod to non-root user before copy to /etc

Reply:
Yup, this will work as WAR. Will explore if there is better way to handle the same.
We can try with selinux policies in hardened OS (if selinux enabled).